spec/protostream_spec.js

```diff
@@ -150,3 +151,36 @@ describe("Put/Get protostream object to/from Infinispan", function () {
     }
     );
 });
+
+describe('Querying in application/x-protostream format',function () {
+    var root = protobuf.parse(myMsg).root;
+    var AwesomeMessage = root.lookupType(".awesomepackage.AwesomeMessage");
+    it("Queries the server", async function (done) {
+        try{
+            var protoMetaClient = await ispn.client(t.local, { authentication: t.authOpts.authentication, cacheName: '___protobuf_metadata', dataFormat: { keyType: "text/plain", valueType: "text/plain" } });
+            var client = await t.client(t.local, { authentication: t.authOpts.authentication, cacheName: 'protoStreamCache', dataFormat: { keyType: "application/x-protostream", valueType: "application/x-protostream" } });
+            await protoMetaClient.put("awesomepackage/AwesomeMessage.proto", myMsg2);
+            await client.clear();
+            for(let i=0;i<10;i++){
+                var payload = { awesomeField: "AwesomeString"+i };
+                var message = AwesomeMessage.create(payload);
+                await client.put(i,message)
+            }
+            var queryResp1 = await client.query({queryString:`from awesomepackage.AwesomeMessage ORDER BY awesome_field`});
+            expect(queryResp1.length).toBe(10);
+            var queryResp2 = await client.query({queryString:`from awesomepackage.AwesomeMessage a where a.awesome_field='AwesomeString1'`});
+            expect(queryResp2.length).toBe(1);
```

Comment:
better to also a little check on the content
```suggestion
            expect(queryResp2.length).toBe(1);
            expect(queryResp2[0].awesomeField).toEqual('AwesomeString1');
```